build.gradle

```diff
@@ -49,6 +49,7 @@ ext {
     playServicesVersion = '16.0.1'
     supportAnnotationsVersion = '28.0.0'
     googleTruthVersion = '0.45'
+    grpcVersion = '1.29.0'
```

Comment:
1.29.0 depends on guava 28.2 

https://github.com/grpc/grpc-java/blob/master/build.gradle#L53 

but we have a hard dependency on 27

https://github.com/firebase/firebase-android-sdk/blob/master/firebase-inappmessaging/firebase-inappmessaging.gradle#L107

I'm curious on how does the tests worked with this issue.